<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_inventaire</class>
 <widget class="QWidget" name="w_inventaire">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>886</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="windowTitle">
   <string>w_inventaire</string>
  </property>
  <widget class="QTableWidget" name="table">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>941</width>
     <height>451</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnViewIssue">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>View issue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInventaire">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Show inventaire</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboYear">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExport">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Export </string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>961</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Current line from inventory</string>
   </property>
   <widget class="QTableWidget" name="tableClickMissing">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>931</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>700</y>
     <width>961</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Issues en trop</string>
   </property>
   <widget class="QTableWidget" name="tableEnTrop">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>931</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>570</y>
     <width>961</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Missing from inventaire</string>
   </property>
   <widget class="QTableWidget" name="tableMissingInventaire">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>931</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnShowErrors">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Show only errors</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
